src/components/Demographics/Demographics.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { shape } from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  StyleSheet, Text, View, SectionList,
+} from 'react-native';
+
+import {
+  getResources,
+  getPatient,
+  getPatientGender,
+  getPatientBirthDate,
+  getPatientAge,
+  getPatientAddresses,
+  renderAddress,
+} from '../../resources/fhirReader';
+import Colors from '../../constants/Colors';
+import mockBundle from '../../../assets/mock_data/bundle-blake-eichmann.json';
+import { clearPatientData } from '../../features/patient/patientDataSlice';
+
+const Demographics = ({
+  patientData,
+}) => {
+  const resources = patientData ? getResources(patientData) : getResources(mockBundle);
+
+  const patient = getPatient(resources);
+  const birthDate = getPatientBirthDate(patient);
+  const age = getPatientAge(patient);
+  const gender = getPatientGender(patient);
+  const address = renderAddress(getPatientAddresses(patient));
+
+  const demographics = [
+    {
+      title: 'Birth date',
+      data: [birthDate],
+    },
+    {
+      title: 'Age',
+      data: [age],
+    },
+    {
+      title: 'Gender',
+      data: [gender],
+    },
+    {
+      title: 'Address',
+      data: [address],
+    },
+  ];
+
+  // eslint-disable-next-line react/prop-types
```

Comment:
Should we add props validation instead of disabling the error?

Reply:
Seems overkill for something nested? It wasn't immediately obvious how to add it to Item - happy to do it but another time